src/routes/embedded/account/backup-shares-reminder/account-backup-shares-reminder.view.tsx

```diff
@@ -34,16 +36,34 @@ export function AccountBackupSharesReminderEmbeddedView() {
 
   return (
     <Card
-      headerIcon={<WarningCircledIcon />}
       headerText="Wallet backup"
       subtitle={"Secure your wallet by backing it up"}
       footerElement={<WanderFooter />}
-      hasBackButton={true}
-      onBackButtonClick={back}
+      hasBackButton={false}
       hasCloseButton={true}
       size="auto"
     >
       <Box>
+        <Text
+          variant="bodySm"
+          style={{
+            marginBottom: 24,
+            marginTop: -16,
+            color: "#0D6CE9",
+            cursor: "pointer"
+          }}
+        >
+          Why should I back up my wallet?
```

Comment:
This should probably be a link or show some information when clicked. You can fix it in a new PR thought.

Reply:
Yes, should be a link but what url should i use for this ?